src/Azure/AzureAD/samples/AzureADB2CSample/wwwroot/js/site.js

```diff
@@ -1 +0,0 @@
-﻿// Write your Javascript code.
```

Comment:
Why does src/Azure/AzureAD/samples/AzureADB2CSample/wwwroot/js/site.min.js still exist❔ I believe this incomplete cleanup applies to a few `*.min.js` files, favicon.ico files, and font files e.g. `src/Azure/AzureAD/samples/AzureADSample/wwwroot/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.*`.

Reply:
What do you mean? They have been removed.

Reply:
Ah, strange. They show up in the GitHub UI as unchanged but don't exist in your branch.